sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentModelAdministrationClient.cs

```diff
@@ -344,21 +370,141 @@ public virtual async Task<Response<DocumentModelDetails>> GetDocumentModelAsync(
         }
 
         /// <summary>
-        /// Deletes the model with the specified model ID.
+        /// Composes a model from a collection of existing models.
+        /// A model built by composition allows multiple models to be called with a single model ID. When a document is
+        /// submitted to be analyzed with its model ID, a classification step is first performed to route it to the correct
+        /// custom model.
         /// </summary>
-        /// <param name="modelId">The ID of the model to delete.</param>
+        /// <param name="waitUntil">
+        /// <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service;
+        /// <see cref="WaitUntil.Started"/> if it should return after starting the operation.
+        /// </param>
+        /// <param name="componentModelIds">List of model ids to use in the composition.</param>
+        /// <param name="modelId">A unique ID for your model. If not specified, a model ID will be created for you.</param>
+        /// <param name="description">An optional description to add to the model.</param>
+        /// <param name="tags">A list of user-defined key-value tag attributes associated with the model.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
-        public virtual Response DeleteDocumentModel(string modelId, CancellationToken cancellationToken = default)
+        /// <returns>
+        /// <para>A <see cref="ComposeDocumentModelOperation"/> to wait on this long-running operation. Its Value upon successful
+        /// completion will contain meta-data about the built model.</para>
+        /// </returns>
+        public virtual async Task<ComposeDocumentModelOperation> ComposeDocumentModelAsync(WaitUntil waitUntil, IEnumerable<string> componentModelIds, string modelId = default, string description = default, IDictionary<string, string> tags = default, CancellationToken cancellationToken = default)
```

Comment:
This method has a large number of parameters. Isn't are max argument list 6?

Reply:
Also curious of the order here. Async is being written before sync, is this what we usually follow?

Reply:
> Also curious of the order here. Async is being written before sync, is this what we usually follow?

As far as I know there's no recommended pattern so every SDK project will have its own. The reason I'm setting the order to async --> sync here is because that's how our other client is written, so we were being inconsistent.

Reply:
No concerns from my side, I was mostly curious to understand if we followed any particular guidance.